### DISCUSSION TITLE
Sign In not showing in New UI ?

### DISCUSSION THREAD

**Discussion by cryptonix21** [blockscout/blockscout#8548] [Support]
NEXT_PUBLIC_IS_ACCOUNT_SUPPORTED=true
NEXT_PUBLIC_AUTH0_CLIENT_ID=
NEXT_PUBLIC_AUTH_URL=
NEXT_PUBLIC_LOGOUT_URL=

After Using these ENV variables the sign-in feature is not showing in the new UI..

**Comment by Mojmir-R** [comment]
Hey @Gopal-Gurram, I would advise you to ask this in our official dev-focused Discord server (discord.gg/blockscout) We have people from the team regularly checking all channels and advising there.
